### ISSUE TITLE
fcc authors page project - step 4 instructions need clarification/rewrite

### ISSUE THREAD

**Issue by hbar1st** [freeCodeCamp/freeCodeCamp#55849]
### Describe the Issue

step 4 of the fcc authors page project (js beta) says:
>The data you get from a GET request is not usable at first. To make the data usable, you can use the .json() method on the Response object to parse it into JSON. If you expand the Prototype of the Response object in the console, you will see the .json() method there.

When we read this statement, because the 2nd sentence follows the first, we assume that it applies to what happens after we use the .json() method but actually it is trying to tell us to look at the console *before* we use the .json() method. (if we try to look up the prototype after we make the code change for the step, we can no longer see it)

To avoid confusion, I suggest rewording this to:
>Before modifying the code, expand the Prototype of the Response object in the console, to see the .json() method there.

As an aside though, people don't know what a Prototype is, so telling them to look it up is not very useful? (We haven't been introduced to Prototypes yet at this point)

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-fetch-and-promises-by-building-an-fcc-authors-page/step-4

### Your code

N/A

### Expected behavior

if we must look up the prototype, it should be stated when to do this (before the code change is completed).
Or we can just skip this instruction completely since we haven't learned what a Prototype is yet?

### System

N/A

**Comment by naomi-lgbt** [comment]
> When we read this statement, because the 2nd sentence follows the first, we assume that it applies to what happens after we use the .json() method but actually it is trying to tell us to look at the console before we use the .json() method. 

I'd argue that's a fallacious assumption, rather than an issue with this step.

> To make the data usable, you can use the .json() method on the Response object to parse it into JSON.

Then:

> If you expand the Prototype of the Response object in the browser console, you will see the .json() method there.

Then:

> Remove console.log(res) and implicitly return res.json() instead.

1. Explains how the data can be made usable
2. Explains how to see the method we're explaining
3. Finally tells the camper to make the change.

> As an aside though, people don't know what a Prototype is, so telling them to look it up is not very useful? (We haven't been introduced to Prototypes yet at this point)

It's literally telling the camper to expand the **property**:

![image](https://github.com/user-attachments/assets/d293bbec-f028-4797-8308-884deaa82cc7)

**Comment by naomi-lgbt** [comment]
In short, I'd argue there's nothing to fix here.

**Comment by hbar1st** [comment]
okay, it is possible I'm the only one who encountered this. I hope it can be revisited if more people have the same concerns in the future.
